.github/workflows/build_and_test.yml

```diff
@@ -162,12 +191,20 @@ jobs:
     - name: Checkout Kong source code
       uses: actions/checkout@v3
 
-    - name: Lookup build cache
-      id: cache-deps
-      uses: actions/cache@v3
+    - name: Download artifacts
+      uses: actions/download-artifact@v3
       with:
-        path: ${{ env.BUILD_ROOT }}
-        key: ${{ needs.build.outputs.cache-key }}
+        name: build
+        path: .
+
+    - name: Extract the artifact contents
+      run: |
+        sudo tar -C /usr/local/ -xzvf artifacts/build.tar.gz
+        sudo mkdir -p ${{ env.BUILD_ROOT }}/kong-dev
+        sudo tar -C ${{ env.BUILD_ROOT }}/kong-dev -xzvf artifacts/build.tar.gz
+
+    - name: Install Kong dev
```

Comment:
are those `Install Kong dev` steps needed?

Reply:
@fffonion It will install some dev-rocks and some binary like `grpcurl`.

Reply:
iirc we already installed the dev rocks on build step for ce (and cached) 🤔

Reply:
```
- name: Build Dev Kong dependencies
      if: steps.cache-deps.outputs.cache-hit != 'true'
      run: |
        make install-dev-rocks
```

This line has been removed in this PR, to keep consistency with EE.